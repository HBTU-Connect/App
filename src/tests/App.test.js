import React from 'react'
import ReactDOM from 'react-dom'
import { mount, shallow } from 'enzyme' 
import TestApp from './TestApp'
// import configureMockStore from 'redux-mock-store'
// import thunkMiddleware from 'redux-thunk'

//configuration 
import {configure} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

// mock store 
// const mockStore = configureMockStore([thunkMiddleware])


it('mounts without crashing', () => {
    mount(<TestApp  />)
});


it('snapshot mached', () => {
  expect(shallow(<TestApp />)).toMatchSnapshot()
})


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<TestApp />, div);
});