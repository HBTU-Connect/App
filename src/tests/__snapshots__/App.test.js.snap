// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot mached 1`] = `ShallowWrapper {}`;
